app/services/cfe/submission_router.rb

```diff
@@ -12,7 +12,7 @@ def call
       if HostEnv.production?
         CFE::SubmissionManager.call(@legal_aid_application.id)
       else
-        CFECivil::SubmissionBuilder.call(@legal_aid_application, save_result: true)
+        CFECivil::SubmissionBuilder.call(@legal_aid_application)
```

Comment:
lack of context info on my part so feel free to ignore, but should this not be doing the same thing on any host now?